.github/workflows/dotnet.yml

```
@@ -0,0 +1,93 @@
+          abp install-libs
+          if [ -f "package.json" ]; then
+            npm install
+            npm run build
+          fi
+        fi
+        
+    - name: Build
+      run: |
+        solution=$(find . -name "*.sln" -type f)
+        dotnet build "$solution" --configuration Release --no-restore
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)